pkgs/development/python-modules/pyscf/default.nix

```diff
@@ -1,57 +1,143 @@
 { lib
 , buildPythonPackage
+, isPy27
 , fetchFromGitHub
 , cmake
 , openblas
 , libcint
 , libxc
+, xcfun
 , h5py
 , numpy
 , scipy
-, pytest
+  # Check Inputs
+, nose
+, nose-exclude
 }:
 
 buildPythonPackage rec {
   pname = "pyscf";
-  version = "1.7.0";
+  version = "1.6.6";
```

Comment:
according to the diff, we are doing `1.7.0` -> `1.6.6`, which doesnt fit the title

Reply:
Yes, that was a mistake. Was testing if lowering version helped.